### ISSUE TITLE
Issue fixed in craft-3 branch, but no release has been issued

### ISSUE THREAD

**Issue by tjdraper** [engram-design/SuperTable#199]
I'm experiencing an issue that has been fixed in the Craft 3 branch by this commit: https://github.com/verbb/super-table/commit/e8dcf38f6829cf0e953d2af0bcf7501e6052764f

But that commit has not made it into a release. I can tell composer to pull from the branch, but I don't like to do that. I prefer to stick to properly tagged releases. Hotfixes such as this would generally be released as soon as possible, no? Without that hotfix, it is impossible to add or edit fields, not just Super Table fields, but any field because it stops the rendering of the add/edit field page.

![screenshot_1](https://user-images.githubusercontent.com/3803475/45055079-774bd500-b054-11e8-89c8-c03aac7e190a.png)

When can a tagged release with this hotfix be expected?

**Comment by nikolowry** [comment]
Until a new release has been pushed, I've changed my `composer.json` to use the `craft-3` branch by adding `https://github.com/verbb/super-table` to `repositories` and changing the version to `dev-craft-3`:

```json
{
  "repositories": [{
    "type": "git",
    "url": "https://github.com/verbb/super-table"
  }],
  "require": {
    "craftcms/cms": "3.0.23",
    "verbb/super-table": "dev-craft-3"
  }
}
```

**Comment by brandonkelly** [comment]
This is a duplicate of #194

@nikolowry there is no need to add the custom repository there; just change the `verbb/super-table` constraint to `"dev-craft-3 as 2.0.8"` and run `composer update`. Craft will automatically get you back on the stable track when v2.0.9 is released and you update from within the Control Panel.

**Comment by nikolowry** [comment]
@brandonkelly thanks -- more of an NPM guy!

**Comment by coryzibell** [comment]
Hey @brandonkelly attempting to update to the newest CraftCMS update (3.0.23.1), I'm getting this error due to the temporary composer constraints:

```
Composer output: Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Problem 1
- The requested package verbb/super-table dev-craft-3 as 2.0.8 exists as verbb/super-table[2.0.8, 2.0.7, 2.0.6, 2.0.5, 2.0.4, 2.0.3, 2.0.2, 2.0.1, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.5, 0.4.0, 2.0.0] but these are rejected by your constraint.
Problem 2
- The requested package verbb/super-table (locked at 2.0.8, required as dev-craft-3 as 2.0.8) is satisfiable by verbb/super-table[2.0.8] but these conflict with your requirements or minimum-stability.
```

Any idea what the best solve is? I have nested matrices and super tables, and want to make sure that field modifications I need to do don't get messed up by not grabbing this Craft update.

Thanks

**Comment by brandonkelly** [comment]
@coryzibell Did you run `composer update` or `composer install` ? Should be the former.

**Comment by coryzibell** [comment]
@brandonkelly ran it on the dev server but forgot to run it locally before the craft update. working now, thanks for your help!

**Comment by engram-design** [comment]
Sorry for the delay all, this should be fixed in [2.0.9](https://github.com/verbb/super-table/releases/tag/2.0.9).
